beater/gcsbeat.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/GoogleCloudPlatform/gcsbeat/beater/codec"
 	"github.com/GoogleCloudPlatform/gcsbeat/beater/storage"
 	"github.com/GoogleCloudPlatform/gcsbeat/config"
-	"github.com/deckarep/golang-set"
+	mapset "github.com/deckarep/golang-set"
```

Comment:
There's a bug in `goimports` that thinks any imports with dashes in them are unused and will remove them unless you give them a name.